packages/ramps-controller/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "paths": {
+      "@metamask/*": ["../*/src"]
+    }
```

Comment:
`paths` is already inside of `tsconfig.packages.json`, do we need it here as well?

``` suggestion
    "baseUrl": "./"
```